rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -160,6 +160,13 @@ void setUDPChannel(UDPChannel udpChannel) {
     }
 
     synchronized void handleMessage(DiscoveryEvent event) {
+        logger.debug("handleMessage - Handling message with " +
+                "state: [{}], " +
+                "type: [{}], " +
+                "networkId: [{}]",
+                state, event.getMessage().getMessageType(),
+                event.getMessage().getNetworkId().isPresent());
```

Comment:
seems there's a small typo here - `.isPresent()` can be omitted

Reply:
Done